ote/src/cljs/ote/ui/main_header.cljs

```diff
@@ -305,8 +307,12 @@
         lang                (get langs current-language "")]
     (str url lang)))
 
-(defn fintraffic-quick-links []
-  [:ul (stylefy/use-style style-topnav/fintraffic-quick-links-menu)
+(defn fintraffic-quick-links [e! app menu-open? desktop?]
+  (js/console.log (str "no onpa notta " menu-open? " ja " desktop? " koska " (:width app) " > " style-base/mobile-width-px))
```

Comment:
Before merging the debug code should either be removed or translated into English.

Reply:
😅 whoops, yeah I'll remove this